test/call-test.js

```diff
@@ -807,8 +805,9 @@
                 assert.equals(object.doIt.getCall(0).toString().replace(/ at.*/g, ""), "doIt() !TypeError(Oh noes!)");
             },
 
+            // these tests are ensuring that call.toString is tightly coupled to sinon.format
```

Comment:
The solution is for the tests to use the commonjs module format to include exactly what it needs instead of the global sinon object.